src/classes/BDI_DataImportService.cls

```diff
@@ -788,92 +861,34 @@ global with sharing class BDI_DataImportService {
     * @param targetObjectName the underlying object the bdi object is (ie, Contact, Account, etc)
     * @param dataImportFields the DataImport fields to consider
     * @return Map<String, String> a map of DataImport fields to underlying object fields (eg, Donation_Amount__c => Amount)
+    *
+    * TODO: Probably deprecate this method in favor of instance variant below
```

Comment:
Yes - we have yet to formally deprecate this method and it would premature to do so, since it's declared global. I would prefer to deprecate it in a future release but the comment still applies.

Reply:
Plus, if I do say so myself, it is an OUTSTANDING to do